api/sql/schema/750$transfer.idle.execute.sql

```diff
@@ -141,19 +143,28 @@ BEGIN TRY
             t.settlementDate issuerSettlementDate,
             t.merchantType,
             e.udfDetails udfAcquirer,
-            er.udfDetails udfAcquirerResponse,
+            er.udfDetails acquirerError,
+            er.type acquirerErrorType,
             t.transferId,
             t.transferIdAcquirer,
+            CASE WHEN t.issuerTxState = 1 THEN 1 ELSE 0 END [issuerTimeout],
             t.sourceAccount,
             t.destinationAccount,
             t.channelType,
-            cin.itemCode
+            cin.itemCode,
+            t.retrievalReferenceNumber,
+            t.reversed,
+            t.reversedLedger,
+            rr.reverseIssuer & (t.reversed ^ 1) reverseIssuer,
+            CASE WHEN rr.reverseLedger = 1 AND t.issuerId != t.ledgerId AND t.reversedLedger = 0 THEN 1 ELSE 0 END reverseLedger
         FROM
             @updated u
         JOIN
             [transfer].[transfer] t ON t.transferId = u.txid
         JOIN
             core.itemName cin ON cin.itemNameId = t.transferTypeId
+        JOIN
+            [transfer].[sReversal] rr ON rr.transferId = u.txid
```

Comment:
I think we can avoid this extra join if we add `reverseIssuer` and `reverseLedger` in `@updated` table